test/fixtures/meals.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
use `FactoryBot` with `Faker` instead of fixtures

[FactoryBot](https://github.com/thoughtbot/factory_bot_rails)
[Faker](https://github.com/stympy/faker)